src/utils/audio-context-fix.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Mobile Safari and Chrome will start Audio contexts in a "suspended" state.
+ * A user interaction (touch/mouse event) is needed in order to resume the AudioContext.
+ */
+
+document.addEventListener("DOMContentLoaded", () => {
+  const ctx = THREE.AudioContext.getContext();
```

Comment:
I see this (other) error in Chrome: related?

> `three.js:38150 Uncaught (in promise) DOMException: Cannot decode detached ArrayBuffer`

![image](https://user-images.githubusercontent.com/203725/39842259-a75a2632-539a-11e8-9847-b840e2bfae81.png)